test/dedup-id/src/test/test.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+const moment = require('abacus-moment');
+const oauth = require('abacus-oauth');
+const request = require('abacus-request');
+const { extend } = require('underscore');
+const util = require('util');
+const uuid = require('uuid');
+
+const debug = require('abacus-debug')('abacus-dedup-id-test');
+
+const doGet = util.promisify(request.get);
+const doPost = util.promisify(request.post);
+
+const systemClientId = process.env.CLIENT_ID;
+const systemClientSecret = process.env.CLIENT_SECRET;
+const authServerURL = process.env.AUTH_SERVER_URL || 'http://localhost:9882';
+const collectorURL = process.env.COLLECTOR_URL || 'http://localhost:9080';
+const reportingURL = process.env.REPORTING_URL || 'http://localhost:9088';
+const pollInterval = process.env.POLL_INTERVAL || 300;
+
+const localMeterURL = 'http://localhost:9100';
+
+describe('dedup acceptance test', () => {
+  const timestamp = moment.now();
+  const testQuantity = 1;
+  const dedupId = 'test-dedup-id';
+  const differentDedupId = 'different-dedup-id';
+
+  let docWithoutDedupId;
+  let docWithDedupId;
+  let orgId;
+
+  let systemToken;
+
+  const secured = () => process.env.SECURED === 'true';
```

Comment:
Why is this `secured` constant inside the `describe` when all the rest environment configurations are outside?
Is there any reason for it to be a function? Do you expect the SECURED environment variable to dynamically change throughout the duration of the test?